packages/@atjson/document/src/annotation.ts

```diff
@@ -12,6 +12,13 @@ function areAttributesEqual(
   lhsAnnotationAttributes: any,
   rhsAnnotationAttributes: any
 ): boolean {
+  let hasUnEqualLengths =
+    Object.keys(lhsAnnotationAttributes).length !==
```

Comment:
`lhsAnnotationAttributes` is type `any` but `Object.keys` is only defined on objects; can you narrow the argument types for this function?